lambda/js/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -26,12 +26,23 @@ import scala.scalajs.js.|
 private[lambda] trait IOLambdaPlatform[Event, Result] {
   this: IOLambda[Event, Result] =>
 
+  /**
+   * Lambda handler. Implement this type with a val and a call to `handlerFn` to export your
+   * handler.
+   *
+   * @example
+   *   {{{
+   *  val handler: HandlerFn = handlerFn
+   *   }}}
+   */
+  final type HandlerFn = js.Function2[js.Any, facade.Context, js.Promise[js.Any | Unit]]
```

Comment:
Does this actually work, if `facade.Context` is package-private?

Reply:
Yes! The type is public, but the types of the `Function2` are not, and I guess that works :)